core/services/workflows/v2/engine.go

```diff
@@ -173,11 +172,18 @@ func (e *Engine) runTriggerSubscriptionPhase(ctx context.Context) error {
 	// call into the workflow to get trigger subscriptions
 	subCtx, subCancel := context.WithTimeout(ctx, time.Millisecond*time.Duration(e.cfg.LocalLimits.TriggerSubscriptionRequestTimeoutMs))
 	defer subCancel()
+
+	userLogChan := make(chan *protoevents.LogLine, e.cfg.LocalLimits.MaxUserLogEventsPerExecution)
+	defer close(userLogChan)
```

Comment:
what is this for? How does this change the beholder logs emitted

Reply:
added because previously there were no beholder logs emitted from logs created during the subscription phase of a workflow